src/utils.js

```diff
@@ -24,6 +24,8 @@ export const IS_TESTING =
 export const IS_PRODUCTION =
   typeof WEBPACK_IS_PRODUCTION === "undefined" ? false : WEBPACK_IS_PRODUCTION;
 
+export const READTHEDOCS_URL_CHANGED_EVENT = "readthedocsUrlChanged";
```

Comment:
```suggestion
export const READTHEDOCS_URL_CHANGED_EVENT = "readthedocs-url-changed";
```

We are using `readthedocs-addons-data-ready` or similar for another event we are triggering.